Resources/Prototypes/Entities/Items/Clothing/cloaks.yml

```diff
@@ -0,0 +1,106 @@
+- type: entity
+  parent: Clothing
+  id: CloakClothing
+  abstract: true
+  name: cloak
+  description:
+  components:
+  - type: Sprite
+    sprite: Clothing/cloak.rsi
+  - type: Icon
+    sprite: Clothing/cloak.rsi
+  - type: Clothing
+    Size: 10
+    QuickEquip: true
+    Slots:
+    - back
+    sprite: Clothing/cloak.rsi
+
+- type: entity
+  parent: CloakClothing
+  id: CapCloak
+  name: captain's cloak
+  description:
+  components:
+  - type: Sprite
+    state: capcloak
+  - type: Icon
+    state: capcloak
+  - type: Clothing
+    HeldPrefix: capcloak
+
+- type: entity
+  parent: CloakClothing
+  id: HosCloak
+  name: security cloak
```

Comment:
Use an acronym or the whole name of the role

Reply:
![image](https://user-images.githubusercontent.com/49448379/83690163-c59e5700-a5df-11ea-8c60-8fb78d516c89.png)